app/middleware/gp.js

```diff
@@ -0,0 +1,86 @@
+const util = require('util');
+const http = require('http');
+const gpDetailsParser = require('../utilities/gpDetailsParser');
+const gpOpeningTimesParser = require('../utilities/gpOpeningTimesParser');
+const daysOfTheWeek = require('../utilities/constants').daysOfTheWeek;
+
+function getDetails(req, res, next) {
+  http.get(req.urlForGp, (response) => {
+    let syndicationXml = '';
+    response.on('data', (chunk) => {
+      syndicationXml += chunk;
+    });
+
+    response.on('end', () => {
+      if (response.statusCode === 200) {
+        /* Disabled since assigning to res is recommended best practice my Express */
```

Comment:
Ultra picky: `my` should be `by`?
And can the wording be changed to `...assigning to the response is...`?